python/lsst/ap/association/ssSingleFrameAssociation.py

```diff
@@ -137,8 +138,11 @@ def run(self,
         RuntimeError
             Raised if duplicate DiaObjects or duplicate DiaSources are found.
         """
-        # Associate DiaSources with DiaObjects
-        associatedSsSources = self.associateSources(sourceTable, solarSystemObjectTable, exposure)
+        if solarSystemObjectTable is None:
+            associatedSsSources = None
```

Comment:
It sounds like this still counts as a success for the purposes of pipeline tracking, so I'd rather raise `NoWorkFound` just to be explicit that the task did nothing.

Either way, we'll have to be careful that downstream tasks take `ssSingleFrameAssociatedSources` as an optional input. 😬